.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Support Request
+about: Ask questions about this project
+
+---
+
+<!-- 
+STOP -- PLEASE READ!
+
+GitHub is not the right place for support requests.
+
+If you're looking for help, post your question on the [Kubernetes Slack ](http://slack.k8s.io/) Sig-Storage Channel.
```

Comment:
```suggestion
If you're looking for help, post your question on the [Kubernetes Slack ](http://slack.k8s.io/) #sig-storage Channel.
```

Reply:
I think the official spelling is "SIG Storage" (capital SIG, no hyphen). At least that was most common among SIGs the last time I checked.